corehq/apps/hqadmin/reports.py

```diff
@@ -1203,21 +1203,21 @@ def rows(self):
         now = datetime.utcnow()
         days = now - timedelta(days=90)
 
-        def get_data():
+        def get_data(domains):
             terms = [
                 AggregationTerm('domain', 'domain'),
                 AggregationTerm('commcare_version', 'form.meta.commcare_version')
             ]
-            query = FormES().submitted(gte=days, lte=now)
+            query = FormES().submitted(gte=days, lte=now).domain(domains)
```

Comment:
Sorry for the late feedback but since you're only interested in the aggregations you should do `.size(0)` to prevent returning all the form docs as well.